eng/Versions.props

```
@@ -25,7 +25,7 @@
       These are used as reference assemblies only, so they must not take a ProdCon/source-build
       version. Insert "RefOnly" to avoid assignment via PVP.
     -->
-    <RefOnlyMicrosoftBuildVersion>17.3.2</RefOnlyMicrosoftBuildVersion>
+    <RefOnlyMicrosoftBuildVersion>17.9.5</RefOnlyMicrosoftBuildVersion>
```

dotnet/runtime uses this earlier version, afaik we need to stay compatible with VS17.8
```suggestion
    <RefOnlyMicrosoftBuildVersion>17.8.3</RefOnlyMicrosoftBuildVersion>
```